Assets/MixedRealityToolkit-SDK/Features/SpatialAwareness/System/Handlers/MixedRealitySpatialAwarenessSurfaceFindingHandler.cs

```diff
@@ -3,13 +3,14 @@
 
 using Microsoft.MixedReality.Toolkit.Core.EventDatum.SpatialAwarenessSystem;
 using Microsoft.MixedReality.Toolkit.Core.Interfaces.SpatialAwarenessSystem.Handlers;
+using UnityEngine;
 
 namespace Microsoft.MixedReality.Toolkit.SDK.SpatialAwarenessSystem.Handlers
 {
     /// <summary>
     /// Class providing the default implementation of the <see cref="IMixedRealitySpatialAwarenessSurfaceFindingHandler"/> interface.
     /// </summary>
-    public class MixedRealitySpatialAwarenessSurfaceFindingHandler : IMixedRealitySpatialAwarenessSurfaceFindingHandler
+    public class MixedRealitySpatialAwarenessSurfaceFindingHandler : MonoBehaviour, IMixedRealitySpatialAwarenessSurfaceFindingHandler
     {
         /// <inheritdoc />
         public void OnSurfaceAdded(MixedRealitySpatialAwarenessEventData eventData)
```

Comment:
```suggestion
        public virtual void OnSurfaceAdded(MixedRealitySpatialAwarenessEventData eventData)
```

I can't add a suggestion for later lines but they need to be updated as well.